app/controllers/static_pages_controller.rb

```diff
@@ -1,15 +1,17 @@
 class StaticPagesController < ApplicationController
-  before_action :authorize, except: [:home]
+  # before_action :authorize, except: [:home]
```

Comment:
remove this, need to call authorize directly in each method to assign @user_data.